lib/ansible/module_utils/docker_common.py

```diff
@@ -741,3 +797,23 @@ def get_legacy_docker_diffs(self):
         '''
         result = [entry['name'] for entry in self._diff]
         return result
+
+
+def clean_booleans_for_docker_api(data):
```

Comment:
How about including `dict` there somehow? Like `clean_dict_booleans_for_docker_api`?

Reply:
"There are only two hard things in Computer Science: cache invalidation and naming things."